src/include/btmem.h

```diff
@@ -10,7 +10,7 @@
 
 /* AUTOMATIC FLAG VALUE GENERATION START */
 #define WT_READ_CACHE 0x0001u
-#define WT_READ_CACHE_LEAF 0x0002u
+#define WT_READ_CACHE_LEAF_NO 0x0002u
```

Comment:
The name here isn't meaningful to me. Could we make it clearer. Does it mean to read leaf pages iff they have overflow records? I'm on board with removing the flag later, but I'd like a better name as well.

Reply:
This flag is not needed any more, I updated the assert that was causing the problem and removed it.